lib/wifi_iot.dart

```diff
@@ -50,6 +61,9 @@ class WiFiForIoTPlugin {
     }
   }
 
+  /// Returns whether the WiFi AP is hidden or not
+  @Deprecated(
+      "This is will only work with < Android SDK 26. It could be make to work for >= Android SDK 29, request at https://github.com/alternadom/WiFiFlutter/issues/134.")
```

Comment:
minor typo in error message: `make` -> `made`

Reply:
Resolved in 7f5909a3b4dd935c522bd3a8888b7840f7ddeadf